commands/core/cli.drush.inc

```diff
@@ -1,25 +1,9 @@
 <?php
 
-use Drush\Log\LogLevel;
-use Drupal\Component\Assertion\Handle;
-use Drush\Psysh\DrushHelpCommand;
-use Drush\Psysh\DrushCommand;
-use Drush\Psysh\Shell;
-
 /**
  * Implements hook_drush_command().
  */
 function cli_drush_command() {
-  $items['core-cli'] = array(
-    'description' => 'Open an interactive shell on a Drupal site.',
-    'remote-tty' => TRUE,
```

Comment:
We are losing this remote-tty declaration. I'm not clear what it does. I do know that we should continue to support `drush @remote php`. Please test that.